include/multigrid_pclomp/multi_voxel_grid_covariance_omp_impl.hpp

```diff
@@ -157,28 +189,33 @@ template<typename PointT>
 int MultiVoxelGridCovariance<PointT>::radiusSearch(const PointT &point, double radius, std::vector<LeafConstPtr> &k_leaves, unsigned int max_nn) const {
   k_leaves.clear();
 
-  // Find neighbors within radius in the occupied voxel centroid cloud
+  // First, search from the grid kdtree to find candidate grids
   std::vector<float> k_sqr_distances;
   std::vector<int> k_indices;
-  int k = kdtree_.radiusSearch(point, radius, k_indices, k_sqr_distances, max_nn);
 
-  // Find leaves corresponding to neighbors
+  // This should be fast, since the number of grids is small
```

Comment:
@SakodaShintaro This should be removed, since it is not correct anymore.

Reply:
Fixed in https://github.com/tier4/ndt_omp/pull/48/commits/0d9be0e96ce2bfcb74868015064fddf1bae4272b